src/report-portal.js

```diff
@@ -128,32 +128,48 @@ class ReportPortal {
     async _startFixturePreTest(time, name = "Before Test") {
         //need to close former fixture
         await this._finishFixture(time);
-
+        let fixtureDescription;
+        let hasAttribute = false;
+        if(name !== "Before Test"){
+    
+          fixtureDescription = `
```

Comment:
add check of \n, and execute if exists in the fixtureDescription, else we can skip